new_project/examples/separate_apps_testnet/README.md

```diff
@@ -0,0 +1,203 @@
+# Roadmap to testnet
+
+## Intro
+
+Synced testnet nodes are a requirement for running the COMIT Node Daemon (cnd) against testnet. 
+In order to make your life easier we provide fully synced testnet nodes for your cnd here:
+
+Note that our testnet nodes might not always be running, but are just started before workshops/events.
+If you have any questions please contact us through the [COMIT community chat on gitter](https://gitter.im/comit-network/community).
+
+This guide assumes that you run both the taker and the maker on the same machine (for the example's sake).
+In a more real world scenario maker and taker would of course be separate people on separate machines.
+
+## Setup
+
+### Run cnd taker
+
+```toml
+[network]
+listen = ["/ip4/0.0.0.0/tcp/9939"]
+
+[bitcoin]
+network = "testnet"
+node_url = "http://bobtimus.coblox.tech:18332"
+
+[ethereum]
+node_url = "http://bobtimus.coblox.tech:8545"
+
+[http_api]
+address = "0.0.0.0"
+port = 8000
+```
+
+Example seed file for the taker:
```

Comment:
```suggestion
Example seed file for the taker (should be in "/tmp/comit/taker/seed.pem":
```

Reply:
Although, unless I'm missing something, it would be better not to mention the seed file at all since it is auto-generated?

Reply:
It would definitely be better if we would not need to mention the seedfile. Since we define the ports of the HTTP API endpoints in the config file, the example could fetch the other cnd's identity at the beginning, then it doesn't need to be hardcoded (and we don't have to care about the seed).